admin_manual/configuration/files/encryption_configuration_quick_guide.rst

```diff
@@ -0,0 +1,77 @@
+====================================
+Encryption Configuration Quick Guide
+====================================
+ 
+Encryption Types
+----------------
+
+ownCloud provides two encryption types:
+
+- **User specific Key:** every user has their own private/public key pairs, and the private key is protected by the user's password.
+
+- **Master Key:** there is only one key (or key pair) and all files are encrypted using that key pair.
```

Comment:
and all files are encrypted --> all files are encrypted

Reply:
The original version seems fine to me.